create_db.py

```diff
@@ -38,11 +38,9 @@ def create_default_user(email, password):
     parser = argparse.ArgumentParser()
     parser.add_argument("email", nargs='?', help="The email for super_admin.", default='')
     parser.add_argument("password", nargs='?', help="The password for super_admin.", default='')
-    parser.parse_args()
-    email = parser.email if hasattr(parser, "email") else ''
-    password = parser.password if hasattr(parser, "password") else ''
+    parse = parser.parse_args()
     with current_app.app_context():
         db.create_all()
         stamp()
-        create_default_user(email, password)
+        create_default_user(getattr(parse, "email"), getattr(parse, "password"))
```

Comment:
It is better to use `parser.get('email')` here instead.
@schedutron Regarding `None`. Please check `create_default_user` method. The way it check will return same effect for empty string and None and None makes much sense than empty string.
Also, if you see the method, if the values are not provided it prompts the user for it so I don't think it is the good idea to provide default value unless we are intended to it.

Reply:
There is no parser.get